R/axis.R

```diff
@@ -89,6 +106,16 @@ axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL) {
 }
 
 draw_axis <- function(object, side.name){
+  # method isn't made for multiple axis calls
+  which.axis <- which(names(object[[side.name]]) == 'axis')
+  if (length(which.axis) > 1){
+    for (axis.i in which.axis){
+      tmp <- object
```

Comment:
this was kind of a hack to deal with multiple axis function calls in one object. tmp is the same object, but w/o the other axis calls, that way I can pass tmp back into the same function and it will be treated the same way a single axis object is. Make sense? This function in general needs some attention, and I just added some new junk to it.